File: OCH_Tao/ABC/B/108.js
const [X1, Y1, X2, Y2] = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
const inc = [X2 - X1, Y2 - Y1];
let rslt = [];
rslt.push(X2 - inc[1]);
rslt.push(Y2 + inc[0]);
rslt.push(rslt[0] - inc[0]);
rslt.push(rslt[1] - inc[1]);
console.log(rslt.join(" "));

File: OCH_Tao/ABC/B/108.py
X1,Y1,X2,Y2 = map(int,input().split())
inc = [X2-X1,Y2-Y1]
rslt = []
rslt.append(X2-inc[1])
rslt.append(Y2+inc[0])
rslt.append(rslt[0]-inc[0])
rslt.append(rslt[1]-inc[1])
print(" ".join([str(i) for i in rslt]))